keras_rcnn/layers/object_detection/_proposal_target.py

```diff
@@ -52,7 +52,13 @@ def call(self, inputs, **kwargs):
         # Convert fixed anchors in (x, y, w, h) to (dx, dy, dw, dh)
         bbox_reg_targets = keras_rcnn.backend.bbox_transform(anchors, gt_boxes)
 
-        return labels, bbox_reg_targets
+        #TODO: Why is bbox_reg_targets' shape (5, ?, 4)? Why is gt_boxes' shape (None, None, 4) and not (None, 4)?
+        bbox_reg_targets = keras.backend.reshape(bbox_reg_targets, (-1, 4))
+        return [labels, bbox_reg_targets]
 
     def compute_output_shape(self, input_shape):
-        return (None,), (None, 4)
+        return [(None,), (None, 4)]
+
+    def compute_mask(self, inputs, mask=None):
```

Comment:
It is trying to call `output_mask` for all outputs. I'm not sure why it isn't required for a single output, but for multiple outputs it seems to be required (https://github.com/fchollet/keras/issues/3061). Perhaps something we should try to fix in Keras.